auth-web/src/util/constants.ts

```diff
@@ -159,7 +159,7 @@ export enum SearchFilterCodes {
 
 export enum PaymentTypes {
     CREDIT_CARD = 'CC',
-    BCOL = 'DRAWDOWN',
+    BCOL = 'BCOL',
```

Comment:
Hope this is correct

Reply:
Pay API expects the payment method as DRAWDOWN